### PR TITLE
Packed Station: A small map for lowpop

### PR DESCRIPTION
**Pull request by retlaw34**
## About The Pull Request
Ports the white sands station version of packedstation, moves genetics to science, adds a chemistry lab in place of old genetics, replaces the tesla with a supermatter, adds a abandoned xenobio sat, and other things

esadfas

This isn't finished yet.

Issues:

todo:
- [x] Arrivals shuttle spawns twice.
- [x] Cargo shuttle doesn't work.
- [x] Escape shuttle is named as ministation escape shuttle, works otherwise.
- [x] Replace TEG with a turbine and fix the wiring.
- [x] ~~Give abandoned sat a atmos set up or even toxins set up.~~
- [x] Give new xenobio sat a toxins set up
- [x] replace red wires
- [x] Replace the good firelocks with crappy 1 tile ones.
- [ ] Figure out how to code.

Port of https://github.com/Whitesands13/Whitesands/pull/540

## Why It's Good For The Game
We are lowpop half the time, and this is a lowpop map i think will work to make the station feel less empty during those hours

## Changelog
:cl: retlaw34, Chiemi
add: Nanotrasen found a old abandoned station in a decaying orbit belonging to them: Ministation. Instead of salvaging it, they fixed it up, renamed it to Packedstation, and called it a day.
fix: ruin didn't spawn oops
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\map_files\PackedStation\PackedStation.dmm"
```
Comment (EgorDinamit) [inline tgstation.dme:17]: orrrange

Comment (retlaw34) [reply tgstation.dme:17]: im feeling-the-pain

**Comment by retlaw34** [comment]
this now works on paper

**Comment by EgorDinamit** [comment]
image

Those are supposed to be pixel_x/y

**Review by EgorDinamit — APPROVED** [review]
Merging in a day if no other issues arise.

**Pull request merged**